manifests/b/BrianApps/Sizer/Dev/4.0.640/BrianApps.Sizer.Dev.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BrianApps.Sizer.Dev
+PackageVersion: 4.0.640
+PackageLocale: en-US
+Publisher: Brian Apps
+PublisherUrl: http://www.brianapps.net/
+PublisherSupportUrl: http://www.brianapps.net/contact.html
+Author: Brian Apps
+PackageName: Sizer
```

Comment:
Not sure if `PackageName: Sizer` is safe as it might cause a matching conflict if 4.0.640 gets released

Reply:
The name in ARP is displaying `Sizer 4.0 (dev 640).` Thoughts on changing it to `Sizer (Dev)`?

Reply:
Yeah i see, its still not going tho as we would have to do `Sizer 4.0 (dev` but i'd say its good enough to do `Sizer (Dev)` so they can upgrade to Dev if they want then get sent to Stable as its realized